### ISSUE TITLE
join-testnet-with-binaries.sh fails to restore priv_validator_key.json

### ISSUE THREAD

**Issue by dhinio52** [axelarnetwork/axelarate-community#270]
**Describe the bug**
`**join-testnet-with-binaries.sh**` script fails to recover using the priv_validator_key.json file specified using `**_--tendermint-key_** `when there is a chain reset or trying to restore after removing $HOME/.axelar_testnet

**To Reproduce**

1. rm -rf $HOME/.axelar_testnet
2.  `./join-testnet-with-binaries.sh  --axelar-core v0.9.6  --validator-mnemonic /brightlystake/valdacct --tendermint-key /brightlystake/privkey.json `
3. No error returned by the script . It restores the validator account using the keys specified but fails to restore privkey.json  
4. The script creates a new key in `$HOME/.axelar_testnet/.core/config/priv_validator_key.json` which is different from the contents of `/brightlystake/privkey.json` specified when starting the script.

**Expected behavior**
The tendermint key specified using the parameter `--tendermint-key`  should be restored to $HOME/.axelar_testnet/.core/config/priv_validator_key.json`

**Screenshots/Log excerpts**
If applicable, add screenshots or excerpts from the node log to help explain your problem.

**Cause of issue**

```
if [ ! -f "$CONFIG_DIRECTORY/genesis.json" ]; then
  "$AXELARD" init "$NODE_MONIKER" --chain-id "$AXELARD_CHAIN_ID" --home "$CORE_DIRECTORY"
  if [ -f "$TENDERMINT_KEY_PATH" ]; then
    cp -f "$TENDERMINT_KEY_PATH" "$CONFIG_DIRECTORY/priv_validator_key.json"
  fi
fi
```

Since the genesis file is downloaded earlier in the script the priv_validator_key.json file is not restored from the specified file.

**Possible solution**

Restore the file earlier in the startup script or skip checking  of non existence of  genesis file.

**Comment by aditya-manit** [comment]
Correct me if I am wrong - but are you trying to import `priv_validator_key.json` from the previous chain ? If you want to import account you just need to import using mnemonic.

`priv_validator_key.json`  from other chain will not work on a new chain afaik

**Comment by dhinio52** [comment]
The issue persists when trying to restore for the same chain as well as discussed in discord

**Comment by ggutoski** [comment]
no longer relevant after #289
